lib/stripe/util.rb

```diff
@@ -104,19 +104,29 @@ def self.custom_method(resource, target, name, http_verb, http_path)
     # ==== Attributes
     #
     # * +data+ - Hash of fields and values to be converted into a StripeObject.
+    # * +params+ - Params for +StripeObject+ like filters used in search that
+    #   will be reused on subsequent API calls.
     # * +opts+ - Options for +StripeObject+ like an API key that will be reused
     #   on subsequent API calls.
-    def self.convert_to_stripe_object(data, opts = {})
+    def self.convert_to_stripe_object(data, params = {}, opts = {})
```

Comment:
This is technically a breaking change since this is in the public interface and people could be using it.

You could flip the order here to get around that I think?

Reply:
Wanted to keep the ordering consistent across all methods in ruby. I might just add a new internal method.